lib/app/components/pages/dashboard/views/dashboard_view.dart

```diff
@@ -19,11 +17,9 @@ class DashboardView extends GetView<DashboardController> {
             child: Obx(
               () => IndexedStack(
                 index: controller.tabService.tabIndex.value,
-                children: const [
-                  TabRootView(initialRoot: Routes.home),
-                  TabRootView(initialRoot: Routes.chat),
-                  TabRootView(initialRoot: Routes.mypage)
-                ],
+                children: controller.navigationInfo.map((info) =>
+                    TabRootView(initialRoot: info.initialRoot)
+                ).toList(),
```

Comment:
```suggestion
            children: const [
              TabRootView(initialRoot: Routes.home),
              TabRootView(initialRoot: Routes.search),  //ここに検索画面用のボタン追加するだけで良い
              TabRootView(initialRoot: Routes.chat),
              TabRootView(initialRoot: Routes.mypage)
            ],
```
すいません、大変申しわけないんですが、
モーダル遷移にする必要はないので前の実装に戻して、コメントの場所に虫眼鏡アイコンボタン追加して欲しいです🙇